RockPaperScissors/RockPaperScissors/main.swift

```diff
@@ -6,5 +6,87 @@
 
 import Foundation
 
-print("Hello, World!")
+enum InputError: Error {
+    case invalidNumber
+}
 
+enum RPS: Int {
+    case exit
+    case scissors
+    case rock
+    case paper
+}
+
+enum GameResult: String {
+    case draw = "비겼습니다!"
+    case win = "이겼습니다!"
+    case lose = "졌습니다!"
+}
+
+func randomComputerHand() -> RPS {
+    guard let computerHand = RPS(rawValue: Int.random(in: 1...3)) else {
+        return randomComputerHand()
+    }
+    
+    return computerHand
+}
+
+func printMenu() {
+    print("가위(1), 바위(2), 보(3)! <종료 : 0> : ", terminator: "")
+}
+
+func receiveUserInput() throws -> Int {
+    guard let input = readLine() else {
+        throw InputError.invalidNumber
+    }
+    
+    guard let inputNumber = Int(input) else {
+        throw InputError.invalidNumber
+    }
+    
+    return inputNumber
+}
```

Comment:
LGTM 👍
```suggestion
func receiveUserInput() throws -> Int {
    guard let input = readLine(), let inputNumber = Int(input) else {
        throw InputError.invalidNumber
    }
    
    return inputNumber
}
```
이렇게 작성할 수도 있을 것 같아요😁

Reply:
훨씬 간략해진 것 같아요! 감사합니다~